packages/mock-block-dock/src/bottom-view/logs-view.tsx

```diff
@@ -0,0 +1,259 @@
+import { Message } from "@blockprotocol/core/dist/esm/types";
```

Comment:
Does message get re-exported? It'd be nice to avoid imports from sub-paths.

Reply:
It wasn't previously. I exported it in this commit [a7d31ff](https://github.com/blockprotocol/blockprotocol/pull/507/commits/a7d31ffc4b008919640cbdfa080a3670e0b96862). Although I will have to update the import path in a follow up